modules/lmdbbackend/lmdbbackend.cc

```diff
@@ -673,6 +673,10 @@ LMDBBackend::LMDBBackend(const std::string& suffix)
     d_random_ids = true;
     d_handle_dups = true;
     LMDBLS::s_flag_deleted = true;
+
+    if (atoi(getArg("shards").c_str()) != 1) {
```

Comment:
Not a fan of atoi due to its lack of error handling

Reply:
agreed, but currently this mirrors usage elsewhere in the file